Lib/sqlite3/test/dbapi.py

```diff
@@ -135,6 +135,25 @@ def test_failed_open(self):
     def test_close(self):
         self.cx.close()
 
+    def test_use_after_close(self):
+        sql = "select 1"
+        cu = self.cx.cursor()
+        res = cu.execute(sql)
+        self.cx.close()
+        self.assertRaises(sqlite.ProgrammingError, res.fetchall)
```

Comment:
Is it ok to call cx.cursor() on a closed connection?

Reply:
No, it'll raise a `sqlite3.ProgrammingError`. I can add a test case for that as well.